core/ide/src/main/java/org/lgna/ik/poser/scene/PoserPicturePlaneInteraction.java

```diff
@@ -156,26 +155,25 @@ private double getSphereRayIntersection(Ray ray, SSphere sSphere) {
 
```

Comment:
perhaps we could just accept that we're USING_MATH_CLASSES and delete the else case?

Reply:
Has your pose fiddling done anything that uses this class?
I am unsure either path has been run in a decade. (Not sure if I am arguing to keep the else clause or delete the whole class.)

Reply:
oh, yeah, i'm in this.  I'll take care of it